common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1572,6 +1573,11 @@ public enum API {
          */
         public static final String BROKER_API_GET_FLIGHTS_PATH = "/brokerApi/getFlights";
 
+        /**
+         * ContentProvider path for retrieving Broker Discovery Metadata.
+         */
+        public static final String BROKER_DISCOVERY_METADATA_RETRIEVAL_PATH = "/brokerElection/brokerDiscoveryMetadataRetrieval";
```

Comment:
````suggestion
        public static final String RETRIEVE_BROKER_DISCOVERY_METADATA_PATH = "/brokerApi/retreiveBrokerDiscoveryMetadata";

Reply:
will rename the variable, but i'll keep the path the same (since this is not part of the broker hosting app API (getAccounts, getFlags/setFlags, etc)